src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -38,4 +38,4 @@ public void addGroup(@RequestBody GroupDto groupDto) {
     public GroupDto updateGroup(@PathVariable Long id, @RequestBody(required = false) GroupDto groupDto) {
         return mapper.mapToGroupDto(service.saveGroup(mapper.mapToGroup(groupDto)));
     }
-}
+}
```

Comment:
Add a newline

Reply:
nie rozumiem zasadności tego requesta, co ma do JDP200202-19/EncjaCartTesty

Reply:
To tylko estetyczna uwaga. Raczej zostawia się pustą linię na końcu.
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file